src/collection.js

```diff
@@ -483,6 +483,16 @@ class Collection {
 
     return this
   }
+
+  /**
+   *Returns the average of all collection items
+   *
+   * @returns {Number}
+   * */
+  async avg () {
+    const sum = this.items.reduce((total, num) => { return total + num }, 0)
+    return sum / this.items.length

```

Comment:
1. Can you please add an empty line before the `return` statement?
2. Can you please use the `this.size()` method instead of `this.items.length`?